src/jit/lowerxarch.cpp

```diff
@@ -4087,6 +4087,18 @@ bool Lowering::SetStoreIndOpCountsIfRMWMemOp(GenTreePtr storeInd)
     }
     m_lsra->clearOperandCounts(indirCandidateChild);
 
+    if (varTypeIsByte(storeInd))
+    {
+        // If storeInd is of TYP_BYTE, set indirOpSources to byteable registers.
+        bool containedNode = (indirOpSource->gtLsraInfo.srcCount == 0) && (indirOpSource->gtLsraInfo.dstCount == 0);
```

Comment:
Here we just need to check dstCount of indirOpSource is zero.  if its dstCount == 0 implies it is marked as contained.